principles.md

```diff
@@ -0,0 +1,33 @@
+# Status Principles
+
+The goal of Status is widespread adoption of the decentralized web. Our challenge is achieving mass adoption while staying true to our principles outlined below.
+
+**I. Liberty**
+We believe in the sovereignty of individuals. As a platform that stands for the cause of personal liberty, we aim to maximize social, political, and economic freedoms. This includes being coercion-resistant.
+
+**II. Censorship resistance**
+We enable free flow of information. No content is under surveillance. We abide by the cryptoeconomic design principle of censorship resistance. Even stronger, Status is an agnostic platform for information.
+
+**III. Security**
+We don't compromise on security when building features. We use state-of-the-art technologies, and research new security methods and technologies to make strong security guarantees.
+
+**IV. Privacy**
+Privacy is the power to selectively reveal oneself to the world. For us, it's essential to protect privacy in both communications and transactions, as well as being a pseudo-anonymous platform. Additionally, we strive to provide the right of total anonymity.
+
+**V. Transparency**
+We strive for complete openness and symmetry of information, and have no border between our core contributors and our community. We are frank about our shortcomings, especially when making short-term tradeoffs in service of our long-term goals.
+
+**VI. Openness**
+The software we create is a public good. It is made available via a free and open source license, for anyone to share, modify and benefit from. We believe in permission-less participation.
+
+**VII. Decentralization**
+We minimize centralization across both the software and the organization itself. In other words, we maximize the number of physical computers composing the network, and maximize the number of individuals who have control over the system(s) we are building.
+
+**VIII. Inclusivity**
+We believe in fair and widespread access to our software, with an emphasis on ease-of-use. This also extends to social inclusivity, permissionless participation, interoperability, and investing in educational efforts.
+
+**IX. Continuance**
+We create software incentivized to continue to exist and improve, without the stewardship of a single entity or any of the current team members.
+
+**X. Resourcefulness**
+We are relentlessly resourceful. As we grow and have ready access to capital, it is our obligation to token holders to fight bureaucracy and inefficiencies. This means solving problems in the most effective way possible at lower economic costs (in terms of capital, time and resources).
```

Comment:
>to fight bureaucracy and inefficiencies
I hope we are talking about fighting bureaucracy and inefficiencies within Status itself, not in the world? Wasn't clear for me at first read :)

Reply:
Indeed, decentralized organization is about to avoid monopoly. If you are big, centralized and very profitable - you are the next target to be taken down by community interested in decreasing costs, more rights or capabilities. "fighting bureaucracy and inefficiencies" is a driver to build dedicated community.

Reply:
> If you are big, centralized and very profitable - you are the next target to be taken down

Sounds like de-kulakisation :)

@maxris, I agree that refactoring of inefficiencies is a good thing to do on all levels. But this is a text of Status principles and I don't think that we as a company are going to work on solving all the bureaucracy and inefficiencies of the world. For me that would sound like a false political slogan.

Reply:
> Sounds like de-kulakisation :)

@vkjr , Skype community decreased the cost of phone calls and communication dramatically, but big monopolies lost a lot money. Not me and not you, nowadays, desire to pay such high rates, as they were before, for phone services to any company.
"de-kulakisation" seems doesn't suite well here, because decentralized community is driven by free will, but not with coercion. We can say "de-kulakisation", but it's exactly what monopolies do with their "users".

Reply:
Changed this to "As we grow and have ready access to capital, it is our obligation to token holders to fight bureaucracy and inefficiencies within the organization." as this seemed to be the minimal change that reflected above